src/main/java/com/github/onsdigital/babbage/search/helpers/SearchHelper.java

```diff
@@ -61,8 +61,8 @@ public static ONSSearchResponse search(ONSQuery queries) {
 
     public static List<ONSSearchResponse> searchMultiple(List<ONSQuery> queries) {
         MultiSearchRequestBuilder multiSearchRequestBuilder = getElasticsearchClient().prepareMultiSearch();
-        for (ONSQuery builder : queries) {
-            SearchRequestBuilder searchRequestBuilder = prepare(builder);
+        for (ONSQuery query : queries) {
+            SearchRequestBuilder searchRequestBuilder = prepare(query);
```

Comment:
👇 can you delete the commented out code please.

Reply:
Done 👍